google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1166,6 +1182,11 @@ public SpannerOptions build() {
         // As we are using plain text, we should never send any credentials.
         this.setCredentials(NoCredentials.getInstance());
       }
+      // Set the number of channels to DEFAULT_CHANNELS if not set
```

Comment:
Please clarify: if what is not set? If the answer is numChannels, please consider adding a better comment, since that would be inferable from code